src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
 
 namespace System.Text.Json
 {
+    public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.Generic.ICollection<System.Text.Json.JsonNode>, System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode>, System.Collections.Generic.IList<System.Text.Json.JsonNode>, System.Collections.IEnumerable
```

Comment:
It should also implement `IReadOnlyList<JsonNode>`.  Unfortunately IList<T> does not.